execute/plugin_functions_test.go

```diff
@@ -1556,7 +1558,7 @@ func Test_truncateObservation(t *testing.T) {
 					withData(make([]byte, 100)),
 				),
 			},
-			maxSize: 1789,
+			maxSize: 1819,
```

Comment:
Will revert it back to try after moving the sizes from the observation as suggested above 👍